xrootd/xrdfs/stat.go

```diff
@@ -48,6 +48,24 @@ type EntryStat struct {
 	Mtime       int64     // Mtime is the last modification time in Unix time units.
 }
 
+// ToEntryStat creates an EntryStat that represents same information as the provided info.
+func ToEntryStat(info os.FileInfo) EntryStat {
```

Comment:
I would rename this to make it look like a "constructor" for `EntryStat`.
_e.g.:_
```go
func EntryStatFrom(info os.FileInfo) EntryStat {
    ...
}
```